src/plugins/intel_gpu/include/intel_gpu/op/swiglu.hpp

```diff
@@ -16,16 +16,26 @@ class SwiGLU : public ov::op::Op {
 public:
     OPENVINO_OP("SwiGLU", "gpu_opset");
 
+    enum GluType {
+        Swish = 0,
+        Gelu,
+        Gelu_Tanh
+    };
+
     SwiGLU() = default;
     /// \brief Constructs an SwiGLU operation.
     ///
     /// \param data Input tensor with data
     /// \param axis The index of an axis in "data" along which to perform the split
     /// \param split_lenghts A list containing the sizes of each output tensor along the split "axis"
     /// \param output_type Output element type
+    /// \param glu_type GLU type, one of Swish, Gelu
```

Comment:
minor
Acutal parameters order and comment order is different.
Actual parameters order ->  ... , glu_type, split_to_glu_idx, output_type
description -> ... output_type, glu_type, split_to_glu_idx

Reply:
Oh... I'll fix it. Thank you!